src/process/loader-shell-runner.ts

```diff
@@ -75,11 +75,13 @@ export class LoaderShellRunner implements IShellRunner {
   }
 
   private hideCursor() {
+    // TODO: Disabled this code for now as it resulted in weird behavior when making use of Cliffy prompts.
```

Comment:
f: I dont think we can disable this savely. This will break CLIs when aborting cliffy with Strg + C (or at least it will make the cursor disappear even after the process terminated, which is horrible).